cmd/vclusterctl/cmd/node/delete.go

```diff
@@ -108,3 +119,20 @@ func (o *DeleteOptions) Run(ctx context.Context, args []string) error {
 	o.Log.Infof("Successfully deleted node %s", nodeName)
 	return nil
 }
+
+func (o *DeleteOptions) pollNodeStatus(ctx context.Context, kubeClient *kubernetes.Clientset, nodeName string, errc chan error) {
+	for range time.Tick(time.Second * 5) {
+		node, err := kubeClient.CoreV1().Nodes().Get(ctx, nodeName, metav1.GetOptions{})
+		if err != nil {
```

Comment:
Just for my understanding. Are we not returning on error here because this would lead to aborting the drain even before `RunNodeDrain` does have a chance to run?